### ISSUE TITLE
sql database not found in project

### ISSUE THREAD

**Issue by mina250** [Nazir-Rizwan/Inventory-System-POS-C-sharp#1]
### sql database not found in project

**Comment by Nazir-Rizwan** [comment]
Yes exactly not found database file but see all parameters in code ...
If you are willing sql file than you need to pay some money 💰

**Comment by mina250** [comment]
how much should I pay ?

**Comment by Nazir-Rizwan** [comment]
Basically This is complete software for business software and I have less
price code for free but only file to have pay

**Comment by mina250** [comment]
give me your email

**Comment by mina250** [comment]
i cant see email is **** only

**Comment by Nazir-Rizwan** [comment]
I can't your understand answer
You share email ...

**Comment by Nazir-Rizwan** [comment]
mr.nazirrizwan@gmail.com

**Comment by Nazir-Rizwan** [comment]
***@***.***

**Comment by mina250** [comment]
mina.mazen250@gmail.com
